apps/server/src/modules/h5p-editor/controller/h5p-editor.controller.ts

```diff
@@ -196,27 +197,21 @@ export class H5PEditorController {
 		return deleteSuccessfull;
 	}
 
-	@Get('/:contentId')
-	async getH5PEditor(@Param() params: GetH5PContentParams, @CurrentUser() currentUser: ICurrentUser): Promise<string> {
+	@Get('/edit/:contentId?')
+	async getH5PEditor(@Param() params: GetH5PEditorParams, @CurrentUser() currentUser: ICurrentUser): Promise<string> {
 		// TODO: Get user language
-		if (params.contentId === 'create') {
-			params.contentId = undefined as unknown as string;
-		}
-		const response = this.h5pEditorUc.getH5pEditor(currentUser, params.contentId, 'de');
+		const response = this.h5pEditorUc.getH5pEditor(currentUser, params.contentId as string, 'de');
 		return response;
 	}
 
-	@Post('/:contentId')
+	@Post('/edit/:contentId?')
```

Comment:
that is not a normal way to handle edit operation
you should split create and update into separate methods, Pls look at controllers in other modules for an example

Reply:
I've split this endpoint into two different endpoints